bmsReceiver.test.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+import json
+from bmsReceiver import bmsReceiver
+
+class BmsStreamingTest(unittest.TestCase):
+    def test_calculat4e_moving_avaerage(self):

```

Comment:
minor: spelling in test function name.
hint: word the function name as a statement of functionality. like `computes_moving_avg_of_valid_readings`